Description: <short Specify location to extra m4 macros, in configure.ac.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 AC_INIT
 AC_CONFIG_SRCDIR([src/liboggz/oggz.c])
+AC_CONFIG_MACRO_DIR([m4])
 
 AC_PREREQ(2.53)
 
